contents/docs/integrate/webhooks/discord.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Discord
+sidebar: Docs
+showTitle: true
+---
+
+> For message formatting instructions, see [this dedicated page](/docs/integrate/webhooks/message-formatting).
+## 1. Create an incoming webhook in Discord 
+
+1. Navigate to the channel where you want to add the webhook and select 'Edit Channel' option from left navigation pane. 
+1. Select 'Integerations' option from the left navigation pane. 
+1. If Creating Webhook for first time, click "Create Webhook" button. 
+1. If You have other webhooks, click "View Webhooks" and now click "New Webhook". 
+1. Give any name you prefer(say PostHog). 
+1. Your preferred channel would be appearing in 'Channel' option, If not select you preferred channel from drop-down menu.
```

Comment:
> Your preferred channel would be appearing in 'Channel' option

This doens't read particularly well or I'm missing some context. Could it be re-written to be clearer?

> , If not select you preferred channel from drop-down menu.

The `If` should not be capitalized here. `you` should be `your`.

Reply:
```suggestion
1. Select the channel where the message should be posted from the drop-down.
```